<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Removename</class>
 <widget class="QDialog" name="Removename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>removename â€” The Dollar Tracker</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>Remove Employee</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Employee ID:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editId"/>
   </item>
   <item>
    <widget class="QPushButton" name="deleteBtn">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="area"/>
   </item>
   <item>
    <widget class="QPushButton" name="closeBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
